python/lsst/pipe/tasks/transformMeasurement.py

```diff
@@ -95,7 +127,7 @@ def __init__(self, measConfig, pluginRegistry, inputSchema, *args, **kwargs):
     def getSchemaCatalogs(self):
         """!Return a dict containing an empty catalog representative of this task's output."""
         transformedSrc = afwTable.BaseCatalog(self.mapper.getOutputSchema())
-        return {'transformedSrc': transformedSrc}
+        return {self.outputType: transformedSrc}
```

Comment:
In the meantime, it might be best to rename this parameter `outputDataset`, as that's what we typically call the named things we can get from the `Butler`, with `type` being reserved for the actually class of the saved thing (e.g. `SourceCatalog` or `ExposureF`).